modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -941,6 +955,18 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro validatePredefinedImage {
+		AssertElementPresent(
+			key_imageName = "${fileName}",
+			locator1 = "FormFields#IMAGE_PREDEFINED_VALUE");
+
+		if (isSet(fieldDescription)) {
+			AssertElementPresent(
+				key_description = "${fieldDescription}",
+				locator1 = "FormFields#IMAGE_FIELD_DESCRIPTION");
+		}
+	}
+
```

Comment:
Hi @leticiachiu, the App Builder folder has been deleted from Master. Can you move this macro to FormFields.macro file and rename it to `validateImagePredefinedValue`? Thanks!